cmd/kubemark/hollow-node.go

```diff
@@ -93,18 +93,20 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 }
 
 func (c *hollowNodeConfig) createClientConfigFromFile() (*restclient.Config, error) {
-	clientConfig, err := clientcmd.LoadFromFile(c.KubeconfigPath)
+	clientConfigs, err := clientcmd.LoadFromFile(c.KubeconfigPath)
 	if err != nil {
 		return nil, fmt.Errorf("error while loading kubeconfig from file %v: %v", c.KubeconfigPath, err)
 	}
-	config, err := clientcmd.NewDefaultClientConfig(*clientConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
+	configs, err := clientcmd.NewDefaultClientConfig(*clientConfigs, &clientcmd.ConfigOverrides{}).ClientConfig()
```

Comment:
It is another design issue. Originally we only have one kubeconfig and it is easy to override. Now we have multiple kubeconfig files. Do we need to identify a particular kubeconfig file to overwrite or overwrite all of them?

Reply:
Don't do overwrite. Just aggregate files or content.